components/overview/ContentSection.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <article class="content-section">
+    <div class="content-section__overview">
+      <h2 class="copy__title">
+        {{ title }}
+      </h2>
+      <p class="content-section__description">
+        {{ description }}
+      </p>
+      <AppLink class="copy__link" :url="link.url">
```

Comment:
Let's use `AppCta` instead. It includes the external-link indicator functionality and it is more consistent with other CTAs across the site.

Reply:
I didn't use `AppCta` because on the Figma design there is no external/internal-link. We should ask @JRussellHuffman before chage it. @JRussellHuffman , what do you think?

Reply:
Solved during the sprint review, we move forward with changing to the `AppCta` for consistency.